_delphi_utils_python/delphi_utils/slack_notifier.py

```diff
@@ -0,0 +1,20 @@
+"""Slack notifier for use in Delphi indicators """
+from slack import WebClient
+from slack.errors import SlackApiError
+
+class SlackNotifier:
+    """Notifies slack channels of messages"""
+    def __init__(self, slack_channel, slack_token):
+        self.slack_channel = slack_channel
+        self.client = WebClient(token = slack_token)
+
+    def post_message(self, blocks):
+        """Post a message to the slack channel"""
+        try:
+            self.client.chat_postMessage(
+                channel=self.slack_channel,
+                blocks=blocks
+            )
+        except SlackApiError as e:
+            # You will get a SlackApiError if "ok" is False
+            assert False, e.response["error"]
```

Comment:
IIUC, this is turning a `SlackApiError` into an `AssertionError` with the same message.  Why not just let the `SlackApiError` go uncaught?

Reply:
Yeah good point. Removed.